sources/models/src/aws-k8s-1.22/mod.rs

```diff
@@ -27,4 +27,5 @@ struct Settings {
     container_registry: RegistrySettings,
     oci_hooks: OciHooks,
     cloudformation: CloudFormationSettings,
+    eks_autoscaling: AutoScalingSettings,
```

Comment:
I would just call this setting `autoscaling` so that it is consistent with the ECS setting.